flow/connectors/kafka/kafka.go

```diff
@@ -45,7 +47,7 @@ func NewKafkaConnector(
 		kgo.SoftwareNameAndVersion("peerdb", peerdbenv.PeerDBVersionShaShort()),
 	)
 	if !config.DisableTls {
-		optionalOpts = append(optionalOpts, kgo.DialTLSConfig(&tls.Config{MinVersion: tls.VersionTLS13}))
+		optionalOpts = append(optionalOpts, kgo.DialTLSConfig(&tls.Config{MinVersion: tls.VersionTLS12}))
```

Comment:
This should be a setting. FedRAMP requires 1.3

Reply:
But this is minimum supported, max supported is still 1.3

Reply:
https://github.com/golang/go/blob/go1.22.2/src/crypto/tls/common.go#L738-L751

Reply:
Yes. Guess we can always switch to an environment variable at some point to block TLS 1.2 if a customer wants that